funnel/assets/js/util.js

```diff
@@ -328,6 +328,36 @@ export const Utils = {
     }
     return headerHeight;
   },
+  loadLangTranslations() {
+    // Instantiate i18n with browser context
+    window.i18n = window.i18n();
+    let lang =
+      window.navigator.userLanguage ||
+      window.navigator.language ||
+      window.navigator.browserLanguage ||
+      window.navigator.userLanguage ||
+      window.navigator.systemLanguage;
+    let langShortForm = lang.substring(0, 2);
+    let translatedLang =
+      window.Hasgeek.config.availableLanguages[langShortForm];
```

Comment:
This is okay for now, when only Hindi is supported, but we will have another one coming soon: English (India) vs English (United States), primarily to change date formatting. This code needs more robust language matching.

Reply:
This may be a good solution: https://stackoverflow.com/questions/28483144/i18n-access-locale-resolution-logic-in-javascript